cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -281,7 +281,7 @@ where
             Ok(yaml) => match serde_yaml::from_slice(&yaml) {
                 Ok(value) => t_vec.push(value),
                 Err(e) => {
-                    warn!("Failed to deserialize file at {:?}: {}", &path, e);
+                    debug!("Failed to deserialize file at {:?}: {}", &path, e);
```

Comment:
This looks like a real issue. IMO it should be in warning. Though I have to check the current logs to see if we see this frequently.

Reply:
this happens when the blocking file is not a conditional-block. Old files without conditional block throw this warning,

Reply:
This will be useful if we get conditional edges which fails to serialize.